generator/internal/connection_impl_generator.cc

```diff
@@ -313,21 +299,17 @@ std::string ConnectionImplGenerator::MethodDefinition(
     return R"""(
 StreamRange<$response_type$>
 $connection_class_name$Impl::$method_name$($request_type$ const& request) {
-  auto& stub = stub_;
-  auto retry = std::shared_ptr<$product_namespace$::$retry_policy_name$ const>(retry_policy());
-  auto backoff = std::shared_ptr<BackoffPolicy const>(backoff_policy());
-
-  auto factory = [stub]($request_type$ const& request) {
+  auto current = google::cloud::internal::SaveCurrentOptions();
```

Comment:
optional nit: you can drop the `google::cloud::` if you want. here and below.

Reply:
Leaving as-is